### ISSUE TITLE
the foreign flag not available 

### ISSUE THREAD

**Issue by krishnasen2401** [amodm/api-covid19-in#16]
is foreign no longer working all data in raw patient is failing

**Comment by amodm** [comment]
Which API?

**Comment by krishnasen2401** [comment]
Raw data api

**Comment by krishnasen2401** [comment]
The used to be a is foreign:true/false/null tag

**Comment by krishnasen2401** [comment]
Sorry is patients trace data according to read me other have it as Raw data

**Comment by amodm** [comment]
Are you referring to this? https://api.rootnet.in/covid19-in/unofficial/covid19india.org

**Comment by krishnasen2401** [comment]
Yes

**Comment by amodm** [comment]
I use the NLP API from @NirantK upstream. @NirantK can you please look into this?

**Comment by NirantK** [comment]
@krishnasen2401 Can you please share the expected JSON format or sample?

Asking because [relationship extractor API](https://github.com/NirantK/coronaIndia/blob/master/relationship_extractor.md) still returns the `is_foreign` flag:
```javascript
{
    "patients": [
        {
            "1": {
                "nationality": [],
                "place_attributes": [
                    {
                        "is_foreign": true,
                        "place": "Italy"
                    }
                ],
                "relationship": [],
                "travel": [
                    "Italy"
                ]
            }
        }
    ]
}
```

**Comment by NirantK** [comment]
Okay, making a guess: @krishnasen2401 do you see some something like:

```javascript
{
    "nationality": [],
    "travel": [],
    "relationship": [],
    "place_attributes": []
}
```
for some patient ids?

If yes ---- that is the case when we don't have information about the patient yet. We do not add empty keys to `place_attributes` list.

**Comment by krishnasen2401** [comment]
Previously this is foreign attributes was used to be in place attributes in
api.rootnet.in/covid19-in/unofficial/covid19indian.org api when I first
imported the table now is not present cause when i check the api because of
data change in graph so I am asking was it removed or not

https://drive.google.com/file/d/12WG6vBPW1xce2AKDiJKcMLMQvNTbEPgi/view?usp=drivesdk

**Comment by amodm** [comment]
@krishnasen2401 can you check now?

**Comment by krishnasen2401** [comment]
It's present now but what happened before.why did it got disappeared??

**Comment by NirantK** [comment]
There was a batching error in processing the results from NLP API, which is fixed now: c0ee0c6 

Thanks for raising this!

**Comment by krishnasen2401** [comment]
Thank you
